src/components/20-molecules/datepicker/index.js

```diff
@@ -729,6 +748,63 @@ class AXADatepicker extends NoShadowDOM {
     // any preexisting error should be cleared
     this.error = null;
   }
+
+  // Switches to the next allowed month, when 'direction' is positive.
+  // Switches to the previous allowed month, when 'direction' is negative.
+  // Has no effect, when there is no next/previous allowed month.
+  showNextMonth(direction = 0) {
```

Comment:
I suggest to change the function signature to (event) { return (offset) => { ....}

You don't use direction as a number, you use exactly 1 bit, the sign bit.

Instead, make it a proper number (offset) and uniformly do

month += offset;
The name is odd, how about navigateMonth?!

Then you can uniformly use this.navigateMonth(+1) resp. this.navigateMonth(-1) in click handlers.

Reply:
I first wanted to do it exactly like that but then I decided against it because not every year is an allowed year. If you take 5 as an offset you expect the output to be 5 months later and I think it would be weird if you suddenly skip 10 years. What do you think about this?